### DISCUSSION TITLE
How to stop dev on Android emulator

### DISCUSSION THREAD

**Discussion by syirilrakhulh** [tauri-apps/tauri#8958] [Q&A]
I use `@tauri-apps/cli 2.0.0-beta.4` and my apps working fine with Solid JS as frontend. When I start running dev with Android emulator and I want to stop by close emulator my terminal does not exit/stop running dev command until I kill the process manually by `sudo killall node`. I also cannot stop by press `ctrl + c`. It's weird. But when I use Xcode Simulator after I close simulator it will stop the terminal.

Maybe I miss something in config? Because it only new project, I create Solis JS first after that I add Tauri to existing repo.

**Comment by FabianLars** [comment]
locking in favor of #8959
